models/gc_nnunet_pancreas/meta.json

```diff
@@ -0,0 +1,129 @@
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Segmentation",
+        "classes": [
+          "veins",
+          "arteries",
+          "pancreas",
+          "pancreatic duct",
+          "bile duct",
+          "cysts",
+          "renal vein"
+        ]
+      },
+      {
+        "type": "Prediction",
+        "valueType": "number",
+        "label": "Pancreatic tumor likelihood",
+        "description": "Pancreatic tumor likelihood map with values between 0 and 1",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "nnUnet ",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 242
+      },
+      "evaluation": {
+        "vol_samples": 361
+      },
+      "public": true,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "Fully Automatic Deep Learning Framework for Pancreatic Ductal Adenocarcinoma Detection on Computed Tomography",
+    "version": "",
+    "devteam": "DIAGNijmegen (Diagnostic Image Analysis Group, Radboud UMC, The Netherlands)",
+    "type": "The models were developed using nnUnet. All models employed a 3D U-Net as the base architecture and were trained for 250.000 training steps with five-fold cross-validation.",
+    "date": {
+      "weights": "2023-06-28",
+      "code": "2022-07-19",
+      "pub": "2022-01-13"
+    },
+    "cite": "Alves N, Schuurmans M, Litjens G, Bosma JS, Hermans J, Huisman H. Fully Automatic Deep Learning Framework for Pancreatic Ductal Adenocarcinoma Detection on Computed Tomography. Cancers (Basel). 2022 Jan 13;14(2):376. doi: 10.3390/cancers14020376. PMID: 35053538; PMCID: PMC8774174.",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [
+      {
+        "title": "Fully Automatic Deep Learning Framework for Pancreatic Ductal Adenocarcinoma Detection on Computed Tomography ",
+        "uri": "https://www.mdpi.com/2072-6694/14/2/376"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/CE-CT_PDAC_AutomaticDetection_nnUnet",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended Use",
+      "text": "This algorithm is intended to be used only on venous-phase CECT examinations of patients with clinical suspicion of PDAC. This algorithm should not be used in different patient demographics.",
+      "references": [],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Analysis",
+      "text": "The study evaluated a medical model's performance for tumor detection by analyzing receiver operating characteristic (ROC) and free-response receiver operating characteristic (FROC) curves, assessing both tumor presence and lesion localization, and compared three configurations using statistical tests and ensemble modeling.",
+      "references": [],
```

Comment:
Please add references and (if possible but highly encouraged) the most important KPIs from the analysis.

Reply:
We'll do that.